app/templates/index.html

```diff
@@ -39,7 +38,7 @@
         <div class="col-md-12 text-center">
           <iframe src="//player.vimeo.com/video/64814620?portrait=0&title=0&byline=0"
                   width="500" height="281" frameborder="0"
-                  webkitallowfullscreen mozallowfullscreen allowfullscreen>
+                  webkitallowfullscreen="true" mozallowfullscreen="true" allowfullscreen>
```

Comment:
I don't know exactly, what these attributes do. But maybe you missed `allowfullscreen=true`?

Reply:
this was maked as an issue by a linter, but judging by: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-allowfullscreen i mgiht just replace it with the newer attribute